src/frontend/src/components/LogOut/index.tsx

```diff
@@ -20,6 +21,7 @@ class LogOut extends React.Component<Props, State> {
     this.props.logOut();
     event.preventDefault();
     event.stopPropagation();
+    window.location.pathname = '';
```

Comment:
Do you know if there's any way (or a "correct" way) to do this through an official React Router API? 

If not, I think we should have a comment here explaining why this update is necessary, since we generally want to avoid modifying global variables at any point in the code.

Reply:
Thanks for catching this! I thought that had merged out when I pulled in the other changes. The use of _History_ takes care of this I've dropped the line and can re-submit.

Tested to make sure the behavior of clearing to the base url when logging out.